library/src/morpheus-master/sass/themes/_dark.scss

```diff
@@ -177,7 +177,7 @@
     --errorBanner-bgcolor: var(--sakai-color-red--darker-6);
     --errorBanner-color: var(--sakai-color-red--lighter-4);
 
-    --progress-bar-bg: var(--infoBanner-bgcolor);
```

Comment:
`--progress-bar-bg` was based on color #092435. In my opinion it's too dark for a dark background.

![PBsakai_color_blue_darker_6](https://user-images.githubusercontent.com/81161239/148520081-93a7d022-470e-4313-b12c-a86b3da6b705.png)